package.json

```diff
@@ -45,7 +45,7 @@
     "test-only": "vitest --run",
     "lint": "eslint '{src,test}/**/*.ts'",
     "preview": "vite preview",
-    "typedoc": "typedoc src/index.ts src/client/nosto.d.ts src/testing/testing.ts"
+    "typedoc": "typedoc src/index.ts src/client/nosto.d.ts src/testing.ts"
```

Comment:
--treatWarningsAsErrors doesn't work here, since src/client/nosto.d.ts errors need to be fixed on client script side